### PR TITLE
Done: Added the lesson 15 homework

### PR DESCRIPTION
**Pull request by max3koz**
Done: Added the lesson 15 homework

### CODE REVIEW

File: lesson_15/rhombus.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+    side: float
+    angle_a: float
+    angle_b: float
+
+    def __init__(self, side: float, angle: float):
+        self.set_side_value(side)
+        self.set_angle_values(angle)
+
+    def set_side_value(self, side_new: float) -> None:
```
Comment (DartBrovsky) [inline lesson_15/rhombus.py]: In this case as you are working with construct methods - it is better to make class fields and set methods private for this class.

**Review by DartBrovsky — APPROVED** [review]
I will approve your solution - only I have 1 comment concern regarding access modifiers in your code.

File: lesson_15/rhombus.py (hunk 1 above)
Comment (max3koz) [reply lesson_15/rhombus.py]: I added private class fields and set methods for Rhombus class in the next commit.

**Review by DartBrovsky — APPROVED** [review]
approved

**Pull request merged**